### ISSUE TITLE
materialized_views_ignore_errors setting does not exist

### ISSUE THREAD

**Issue by bx2** [ClickHouse/ClickHouse#48069]
**Describe the issue**

The documentation on materialized views, states that you can provide a setting to the INSERT query that will prevent the write failing in case of any of the materialized view fails. Adding this settings to the query throws an error though:

```
2023.03.27 17:01:20.328772 [ 905726 ] {0b07457a-7315-476f-a39a-b289b13bffef} <Error> TCPHandler: Code: 115. DB::Exception: Unknown setting materialized_views_ignore_errors. (UNKNOWN_SETTING), Stack trace (when copying this message, always include the lines below):

0. DB::Exception::Exception(DB::Exception::MessageMasked&&, int, bool) @ 0xe0c86b5 in /usr/bin/clickhouse
1. ? @ 0x893f7c4 in /usr/bin/clickhouse
2. DB::BaseSettingsHelpers::throwSettingNotFound(std::__1::basic_string_view<char, std::__1::char_traits<char>>) @ 0x1266e5e8 in /usr/bin/clickhouse
3. ? @ 0x123fae80 in /usr/bin/clickhouse
4. DB::SettingsConstraints::checkImpl(DB::Settings const&, DB::SettingChange&, DB::SettingsConstraints::ReactionOnViolation) const @ 0x12520682 in /usr/bin/clickhouse
5. DB::InterpreterSetQuery::executeForCurrentContext() @ 0x13661040 in /usr/bin/clickhouse
6. ? @ 0x1398adc8 in /usr/bin/clickhouse
7. ? @ 0x1397d68b in /usr/bin/clickhouse
8. DB::executeQuery(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, std::__1::shared_ptr<DB::Context>, bool, DB::QueryProcessingStage::Enum) @ 0x1397c9ad in /usr/bin/clickhouse
9. DB::TCPHandler::runImpl() @ 0x14703d39 in /usr/bin/clickhouse
10. DB::TCPHandler::run() @ 0x14719259 in /usr/bin/clickhouse
11. Poco::Net::TCPServerConnection::start() @ 0x1761d1b4 in /usr/bin/clickhouse
12. Poco::Net::TCPServerDispatcher::run() @ 0x1761e3db in /usr/bin/clickhouse
13. Poco::PooledThread::run() @ 0x177a5767 in /usr/bin/clickhouse
14. Poco::ThreadImpl::runnableEntry(void*) @ 0x177a319d in /usr/bin/clickhouse
15. start_thread @ 0x7ea7 in /usr/lib/x86_64-linux-gnu/libpthread-2.31.so
16. clone @ 0xfca2f in /usr/lib/x86_64-linux-gnu/libc-2.31.so
```

I tried both HTTP (in query params and as a SETTTINGS block) and native TCP.

**Additional context**

section "materialized views" of https://clickhouse.com/docs/en/sql-reference/statements/create/view#materialized-view

clickhouse version: 

```
SELECT version()

Query id: ef172caa-72af-4f72-ad3d-3263fae0345f

┌─version()─┐
│ 23.2.4.12 │
└───────────┘
```

**Comment by den-crane** [comment]
Documentation reflect master branch of Clickhouse.

materialized_views_ignore_errors will be available in several days in the 23.3 release.

**Comment by bx2** [comment]
That's fine, but is there a way to view the documentation for the actual, released, and stable versions of clickhouse?
